programs/validator-history/idl/validator_history.json

```diff
@@ -797,7 +797,7 @@
                 "type": "u8"
               },
               {
-                "name": "num_addrs",
+                "name": "numAddrs",
```

Comment:
this shouldn't change as long as we're still on anchor 0.28, maybe the solana program test compatibility is fixed with the new way we're loading in programs but we'd still need to bump versions and make the code compatible